tfhe/examples/utilities/print_doc_bench_parameters.rs

```diff
@@ -0,0 +1,84 @@
+use tfhe::keycache::NamedParam;
+use tfhe::shortint::parameters::classic::gaussian::p_fail_2_minus_64::ks_pbs::{
+    PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64, PARAM_MESSAGE_2_CARRY_2_KS_PBS_GAUSSIAN_2M64,
+    PARAM_MESSAGE_3_CARRY_3_KS_PBS_GAUSSIAN_2M64, PARAM_MESSAGE_4_CARRY_4_KS_PBS_GAUSSIAN_2M64,
+};
+use tfhe::shortint::parameters::classic::tuniform::p_fail_2_minus_64::ks_pbs::PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+use tfhe::shortint::parameters::compact_public_key_only::p_fail_2_minus_64::ks_pbs::PARAM_PKE_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+use tfhe::shortint::parameters::key_switching::p_fail_2_minus_64::ks_pbs::PARAM_KEYSWITCH_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+use tfhe::shortint::parameters::multi_bit::p_fail_2_minus_64::ks_pbs_gpu::{
+    PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
```

Comment:
Same here I would remove `_GAUSSIAN` from the name

Reply:
we'll probably remove some precisions in TUniform, so I think I'll keep it like this for now

Reply:
I'll have to update the PR to set TUniform parameters as default then, once this is merged.

Reply:
yep let's hold on the switch to be able to complete the 0.8 release tasks